nuxeo-multi-tenant-jsf/pom.xml

```diff
@@ -0,0 +1,55 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.nuxeo.multi.tenant</groupId>
+    <artifactId>nuxeo-multi-tenant</artifactId>
+    <version>8.10-SNAPSHOT</version>
+  </parent>
+
+  <groupId>org.nuxeo.multi.tenant</groupId>
+  <artifactId>nuxeo-multi-tenant-jsf</artifactId>
+  <name>Nuxeo Multi Tenant JSF</name>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.nuxeo.multi.tenant</groupId>
+      <artifactId>nuxeo-multi-tenant-core</artifactId>
+      <version>${project.version}</version>
+    </dependency>
```

Comment:
project.version should not be needed here since you added the core module to the parent pom dependency management